middleware/database/migrations/2023_10_16_201340_create_student_mapping.php.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /*
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('student_mappings', function (Blueprint $table) {
+            $table->id();
+            $table->foreignId('student_id')->constrained();
+            $table->foreignId('provider_user_id')->constrained();
```

Comment:
These are no longer coming from our database, but from an external source.
Also, we should have a larger conversation about what type we should use for external ids (also used in ProviderContent). These should probably be strings to support both integer ids and string uuids.